### PR TITLE
Pre Quote Tabs (Buy / Sell)

### PR DESCRIPTION
**Pull request by nalancer08**
### Type of change

- [ ] Chore
- [X] Feature
- [ ] Bug fix

### Linked issue

- [X] Not tracked

### Why

The new design of Faramerz includes some Tabs !!

### Evidence

imagen

### CODE REVIEW

File: SDKAndroid/src/main/java/app/cybrid/sdkandroid/components/ListPricesView.kt
Comment (jc-cybrid) [inline SDKAndroid/src/main/java/app/cybrid/sdkandroid/components/ListPricesView.kt:336]: Not sure if you have a linter for this, but I think it would be nice to have a space after the `:` 
```suggestion
                    index: Int, selectedIndex: Int,
```

File: SDKAndroid/src/main/java/app/cybrid/sdkandroid/components/ListPricesView.kt
Comment (jc-cybrid) [inline SDKAndroid/src/main/java/app/cybrid/sdkandroid/components/ListPricesView.kt:170]: Same here
```suggestion
    onClick: (asset: AssetBankModel, pairAsset: AssetBankModel) -> Unit) {
```

**Review by jc-cybrid — COMMENTED** [review]
Left some Nit comments

**Review by brent-cybrid — APPROVED** [review]
Looks good to me, but please wait for @jc-cybrid to approve since he provided feedback.

**Review by jc-cybrid — APPROVED** [review]
LGTM

**Pull request merged**
